code/modules/antagonists/eldritch_cult/eldritch_magic.dm

```diff
@@ -0,0 +1,395 @@
+	charge_max = 350
+	clothes_req = FALSE
+	invocation = "CL'VE"
+	invocation_type = "whisper"
+	range = 9
+	action_icon = 'icons/mob/actions/actions_ecult.dmi'
+	action_icon_state = "ash_shift"
+	action_background_icon_state = "bg_ecult"
+
+/obj/effect/proc_holder/spell/pointed/ash_cleave/cast(list/targets, mob/user)
+	if(!targets.len)
+		to_chat(user, "<span class='warning'>No target found in range!</span>")
+		return FALSE
+	if(!can_target(targets[1], user))
+		return FALSE
+
+	for(var/mob/living/carbon/human/C in range(1,targets[1]))
+		targets += C
+
+	for(var/X in targets)
+		var/mob/living/carbon/human/target = X
+		if(target == user)
+			continue
+		if(target.anti_magic_check())
+			to_chat(user, "<span class='warning'>The spell had no effect!</span>")
+			target.visible_message("<span class='danger'>[target]'s veins flash with fire, but their magic protection repulses the blaze!</span>", \
+							"<span class='danger'>Your veins flash with fire, but your magic protection repels the blaze!</span>")
+			continue
+
+		target.visible_message("<span class='danger'>[target]'s veins are shredded from within as an unholy blaze erupts from their blood!</span>", \
+							"<span class='danger'>Your veins burst from within and unholy flame erupts from your blood!</span>")
+
+		target.bleed_rate += 8
+		target.adjustFireLoss(10)
+
+/obj/effect/proc_holder/spell/pointed/ash_cleave/can_target(atom/target, mob/user, silent)
+	. = ..()
+	if(!.)
+		return FALSE
+	if(!istype(target,/mob/living/carbon/human))
+		if(!silent)
+			to_chat(user, "<span class='warning'>You are unable to cleave [target]!</span>")
+		return FALSE
+	return TRUE
+
+/obj/effect/proc_holder/spell/pointed/ash_cleave/long
+	charge_max = 650
+
+/obj/effect/proc_holder/spell/targeted/touch/mad_touch
+	name = "Touch of madness"
+	desc = "Touch spell that drains your enemies sanity."
+	hand_path = /obj/item/melee/touch_attack/mad_touch
+	school = "evocation"
+	charge_max = 150
+	clothes_req = FALSE
+	invocation = "OP'N Y'R M'ND"
+	invocation_type = "whisper"
+	action_icon = 'icons/mob/actions/actions_ecult.dmi'
+	action_icon_state = "ash_shift"
+	action_background_icon_state = "bg_ecult"
+
+
+/obj/item/melee/touch_attack/mad_touch
+	name = "Ash Leech"
+	desc = "A sinister looking aura that distorts the flow of reality around it."
+	icon_state = "disintegrate"
+	item_state = "disintegrate"
+	catchphrase = "M'DNESS"
+
+/obj/item/melee/touch_attack/mad_touch/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
+	..()
+	if(ishuman(target))
+		var/mob/living/carbon/human/tar = target
+		if(tar.anti_magic_check())
+			tar.visible_message("<span class='danger'>Spell bounces off of [target]!</span>","<span class='danger'>The spell bounces off of you!</span>")
+			return
+	if(iscarbon(target))
+		var/mob/living/carbon/C = target
+		if(!C.mind.has_antag_datum(/datum/antagonist/ecult))
+			SEND_SIGNAL(C, COMSIG_ADD_MOOD_EVENT, "gates_of_mansus", /datum/mood_event/gates_of_mansus)
```

Comment:
hurray, your target will become depressed in a couple of minutes

in the mean time, they're gonna beat the shit out of you, call you out, and/or call for help, because your dumb ass walked up to them (to melee range!) with an incredibly unsubtle glowing hand and used your first strike to hurt their feely weelies

Reply:
should i make this also cause jittering , confusion and vomiting?

Reply:
no, otherwise it'd be much less subtle

this should be a spell that you cast on someone subtly pre-gank, sort of like changeling stings

Reply:
ah alright